internal/venice-client-common/src/main/java/com/linkedin/venice/controllerapi/D2ServiceDiscoveryResponse.java

```diff
@@ -5,6 +5,7 @@
 
 public class D2ServiceDiscoveryResponse extends ControllerResponse {
```

Comment:
Since this class is used in controllers and all our clients, we should include `[controller]`, `[router]`, `[tc]`, `[fc]`, `[dvc]`, `[samza]` in the PR title and the commit message when merging